TCD-1564: Announcement Bar

spider-sauce opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->
Added announcement bar feature with styling.

- Announcement bar convention in `docusaurus.config.js`
- Styling details in `announcement-bar.css`

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
To create a feedback loop with our customers and also to announce upcoming changes.
### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
<!--- Provide a general summary of your changes in the Title above -->

kimsauce reviewed (commented):
Hi @spider-sauce , I just pushed my changes -- let me know what you think. I tried a few ways to inject the SauceBot image; making it a css button was the only way I could get it to work.

spider-sauce commented:
@kimsauce Looks good to me! 👍 , I fixed a couple of small things:

- fixed vertical alignment on the image (it was causing the text to wrap to the bottom)
- fixed the close button (you couldn't see it because the css styles weren't matching)
- removed the box shadow because it was causing inconsistent behavior based on the screen size.
